src/wizard/boards.ts

```diff
@@ -8,6 +8,7 @@ export const boardSelectOptions = html`
     <option value="lolin_d32_pro">Wemos Lolin D32 Pro</option>
     <option value="featheresp32">Adafruit ESP32 Feather</option>
     <option value="m5stack-core-esp32">M5Stack Core ESP32</option>
+    <option value="seeed_xiao_esp32c3">Seeed Studio XIAO ESP32C3</option>
```

Comment:
Can you add it to the "Other ESP32s" section.

Reply:
Can I add to the start of "Other ESP32s" section?

Reply:
Also, will it use the correct "variant" as "esp32c3"?

Reply:
> Can I add to the start of "Other ESP32s" section?

We're not a marketing tool. Let's sort this alphabetically.

Maybe we should break out the C3.

Reply:
> Maybe we should break out the C3.

Do you mean to have a new section for ESP32C3?

It seems there is a number of ESP32C3 boards available already.

![1666369018211](https://user-images.githubusercontent.com/20147381/197242185-00f4f162-c033-4dc0-9b2f-18d0acca7616.png)

Reply:
Yeah that would probably make more sense.

Reply:
Hi. Sorry for late response.

I have added all the ESP32C3 boards under "ESP32C3" section